src/check/parse/IR.zig

```diff
@@ -31,6 +31,10 @@ tokens: TokenizedBuffer,
 store: NodeStore,
 errors: []const Diagnostic,
 
+// stores the offsets of newlines in the source code
+// only used for pretty-printing the IR toSExpr
+newlines: ?std.ArrayList(u32),
+
```

Comment:
I think we should avoid storing this on the parse IR. Just to keep debug info separate and avoid someone accidentally generating or using it.

Reply:
We'd need an extra arg to pass into all the toSExpr functions, but we can do that.

Reply:
Done